.github/workflows/validate-action-typings.yml

```
@@ -0,0 +1,14 @@
+name: Validate action typings
+
+on:
+  push:
+    branches: [master]
+  pull_request:
```

Lol, I missed the `pull_request` trigger as I'm used to having it trigger only on PRs to default branch

```suggestion
  pull_request:
    branches: [master]
```